t_ja5/test_ja5t.py

```diff
@@ -0,0 +1,250 @@
+"""Functional tests of ja5 filtration."""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+from framework.deproxy_client import DeproxyClientH2
+from framework.deproxy_server import StaticDeproxyServer
+from helpers import dmesg, tf_cfg
+from test_suite import marks, tester
+
+DEPROXY_CLIENT_SSL = {
+    "id": "deproxy",
+    "type": "deproxy",
+    "addr": "${tempesta_ip}",
+    "port": "443",
+    "ssl": True,
+}
+
+DEPROXY_CLIENT_H2 = {
+    "id": "deproxy",
+    "type": "deproxy_h2",
+    "addr": "${tempesta_ip}",
+    "port": "443",
+    "ssl": True,
+}
+
+DEPROXY_SERVER = {
+    "id": "deproxy",
+    "type": "deproxy",
+    "port": "8000",
+    "response": "static",
+    "response_content": "HTTP/1.1 200 OK\r\nConnection: keep-alive\r\nContent-Length: 0\r\n\r\n",
+}
+
+# Number of open connections
+CONCURRENT_CONNECTIONS = int(tf_cfg.cfg.get("General", "concurrent_connections"))
+# Number of threads to use for wrk and h2load tests
+THREADS = int(tf_cfg.cfg.get("General", "stress_threads"))
+
+# Number of requests to make
+REQUESTS_COUNT = int(tf_cfg.cfg.get("General", "stress_requests_count"))
+# Time to wait for single request completion
+DURATION = int(tf_cfg.cfg.get("General", "duration"))
+
+
+class TestJa5tBase(tester.TempestaTest):
+    tempesta = {
+        "config": """
+listen 443 proto=h2,https;
+
+server ${server_ip}:8000;
+
+frang_limits {http_strict_host_checking false;}
+
+tls_certificate ${tempesta_workdir}/tempesta.crt;
+tls_certificate_key ${tempesta_workdir}/tempesta.key;
+tls_match_any_server_name;
+""",
+    }
+
+    backends = [DEPROXY_SERVER]
+
+    def setup_test(self):
+        self.start_all_services()
+
+        srv: StaticDeproxyServer = self.get_server("deproxy")
+        srv.set_response(
+            "HTTP/1.1 200 OK\r\n"
+            + "Connection: keep-alive\r\n"
+            + "Content-Length: 13\r\n"
+            + "Content-Type: text/html\r\n"
+            + "\r\n"
+            + "<html></html>"
+        )
```

Comment:
This is an extra code. Just set the response in the server config and remove this method. Also you can remove this class. Just create a new variable - TEMPESTA_CONF